e2e/interchaintestv8/ibc_eureka_test.go

```diff
@@ -855,30 +837,38 @@ func (s *IbcEurekaTestSuite) ICS20TransferNativeCosmosCoinsToEthereumAndBackTest
 			ProofCommitment: ucAndMemProof,
 			ProofHeight:     *proofHeight,
 		}
-
 		tx, err := s.ics26Contract.RecvPacket(s.GetTransactOpts(s.key), msg)
 		s.Require().NoError(err)
 		receipt := s.GetTxReciept(ctx, eth, tx.Hash())
 		s.Require().Equal(ethtypes.ReceiptStatusSuccessful, receipt.Status)
 		recvBlockNumber = receipt.BlockNumber.Int64()
-
 		if s.generateFixtures {
 			s.Require().NoError(types.GenerateAndSaveFixture(fmt.Sprintf("receiveNativePacket-%s.json", pt.String()), s.contractAddresses.Erc20, "recvPacket", msg, packet))
 		}
 
+		// We removed this event, so we have to find a way to read the contract address that has been generated (or find) on ethereum when cosmos coin are received
```

Comment:
I think this option is fine. Having a way to look up those addresses seems more than reasonable.

So, you can remove this comment since you are doing this below 👍